plugin.audio.sverigesradio/resources/language/resource.language.en_gb/strings.po

```diff
@@ -0,0 +1,115 @@
+# XBMC Media Center language file
```

Comment:
please change to /resources/language/English/strings.po  for backward compatibility with gotham

Reply:
Done